### ISSUE TITLE
[Svelte@5] Knip fails to resolve `*.svelte.(j|t)s` files

### ISSUE THREAD

**Issue by vojtechsimetka** [webpro/knip#651]
## Description
Svelte v5 introduces `stores` that live in `*.svelte.(j|t)s` files. Importing such files is not recognised by knip. Svelte 5 is in RC state right now, there is a call to prepare tooling to work with v5 as it is highly unlikely there will be any breaking change.

Neither of these imports work:
```
import * from '.foo.svelte`
import * from '.foo.svelte.ts`
import * from '.foo.svelte.js`

import * from '.foo.svelte?raw`
import * from '.foo.svelte.ts?raw`
import * from '.foo.svelte.js?raw`
```

## Minimal reproducible example
https://github.com/vojtechsimetka/knip-svelte-5-unresolved-imports

## Error
```sh
> bunx --bun knip

Analyzing workspace ....
4:48:32 PM [vite-plugin-svelte] Your are using Svelte 5.0.0-next.1. Svelte 5 support is experimental, breaking changes can occur in any release until this notice is removed.
work in progress:
 - svelte-inspector is disabled until dev mode implements node to code mapping
 - hmr for .svelte files is disabled until hmr api is implemented
Unused files (1)
src/lib/store.svelte.ts
Unresolved imports (2)
$lib/store.svelte         src/lib/component.svelte:1:9
$lib/store.svelte.ts?raw  src/routes/+page.svelte:3:18
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by webpro** [comment]
Is it something you can add manually to paths config?

Like here: https://knip.dev/features/compilers#svelte

If so, we can add it to the docs.

**Comment by vojtechsimetka** [comment]
The problem is not the `$lib` alias but the `.svelte.ts` suffix.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.17.0-canary.1. See [Release 5.17.0-canary.1](https://github.com/webpro/knip/releases/tag/5.17.0-canary.1) for release notes._

_Using Knip in a commercial project? Please consider [sponsoring me](https://github.com/sponsors/webpro)._

**Comment by webpro** [comment]
Sorry, was too fast answering via mobile.

Should be better now, but note that it's still a `canary` version.
